config/management/genesis/src/command.rs

```diff
@@ -374,17 +370,18 @@ pub mod tests {
 
         // Verify the validator config in the transaction has the correct account address
         match uploaded_user_transaction.payload() {
-            TransactionPayload::Script(script) => {
-                assert_eq!(4, script.args().len());
-
-                match script.args().get(0).unwrap() {
-                    TransactionArgument::Address(account_address) => {
-                        assert_eq!(&owner_account, account_address);
-                    }
-                    _ => panic!(
-                        "Found an invalid argument type for the validator-config transaction script!"
-                    ),
-                };
+            TransactionPayload::ScriptFunction(script_function) => {
+                assert_eq!(4, script_function.args().len());
+
+                // TODO review how do I decode this?
+                // match script_function.args().get(0).unwrap() {
+                //     TransactionArgument::Address(account_address) => {
```

Comment:
@vgao1996 any thoughts?

Reply:
Try `match MoveValue::simple_deserialize(..., &MoveTypeLayout::Address)`?

Reply:
You should be able to `BCS::from_bytes::<AccountAddress>(script_function.args().get(0).unwrap()` to get an `AccountAddress` if I understand what's going on here correctly.